package.json

```diff
@@ -13,10 +13,18 @@
     "dev": "nodemon",
     "build:clean": "rm -rf build",
     "build": "yarn build:clean && yarn tsc",
+    "start": "node ./build/index.js",

```

Comment:
Shouldn't the script "start" be `yarn build && node ./build/index.js`?

Reply:
I think that it is better to separate this two command so that is more patronized with other frameworks that I've used.

Reply:
So when you copy your project inside docker container your `/builder` go together ? Beucase if not, your container won't work. The pattern that you are used is in `Javascript` right? So you simply run the index.js file, but in `Typescript` you need to transpiler the files, so that's why we build before running the index, because if you forgot to build before copying the files to the container you will run an old version of your app.

Reply:
I the Dockerfile I have this code:
```
RUN yarn build

CMD ["yarn", "start"]
```

So I run the build before starting.